libs/model-assessment/src/lib/ModelAssessmentDashboard/Controls/ModelOverview/ObjectDetectionModelOverview.tsx

```diff
@@ -96,7 +97,12 @@ export class ObjectDetectionWidgets extends React.PureComponent<IObjectDetection
           valueFormat={(value: number): string => `IoU=${value}%`}
           showValue
         />
-      </Stack.Item>
+        <InfoCallout
```

Comment:
this looks a bit far out, maybe you should put it next to the text/label instead?

Reply:
Curious on how I should do that? It's displayed to the right of the widget, and still close to `IoU=x%`.

I also tried displaying it below the IoU threshold:

![image](https://user-images.githubusercontent.com/37190647/233098854-ad4c5934-62eb-4df4-924e-c974b37d9af2.png)

Let me know your preference or if there's a way to still bring it closer to the slider title. Thanks!